### DISCUSSION TITLE
When using wrap as bundle, how can I change the name?

### DISCUSSION THREAD

**Discussion by git-sources** [eclipse-tycho/tycho#3245] [Q&A] [answered]
I create a jar using BND but how can I convert it into a bundle? I used wrapAsBundle in the target platform but it changes the name. I would like to have the same name. So can I configure it or create the bundle manually? One question in addition, I would like to sign the jar using maven execute plug-in but it gets always installed before.

**Comment by laeubi** [comment]
> I create a jar using BND but how can I convert it into a bundle?

Bnd can already create bundles so it is a bit unclear, maybe you can share an example?

**Reply by git-sources** [reply]
The bundle is created and it is wrapped but with  groupid.groupid.artifactId . I just want the artifact as name. Instead of "com.mygroup.com.mygroup.com.google.guava" I would like to have just "com.google.guava".

**Comment by git-sources** [comment, marked as answer]
I found the issue, the parent artifact Id had a typo. Now it's working. Strange behavior.
